hnn_core/tests/test_io.py

```diff
@@ -0,0 +1,354 @@
+
+# Authors: George Dang <george_dang@brown.edu>
+#          Rajat Partani <rajatpartani@gmail.com>
+
+from pathlib import Path
+from numpy.testing import assert_allclose
+from h5io import write_hdf5, read_hdf5
+import pytest
+import numpy as np
+
+from hnn_core import (read_network, simulate_dipole, read_params,
+                      jones_2009_model, calcium_model,
+                      )
+
+from hnn_core.hnn_io import (_cell_response_to_dict, _rec_array_to_dict,
+                             _connectivity_to_list_of_dicts,
+                             _external_drive_to_dict, _str_to_node
+                             )
+
+hnn_core_root = Path(__file__).parents[1]
+assets_path = Path(hnn_core_root, 'tests', 'assets')
+
+
+@pytest.fixture
+def params():
+    params_path = Path(hnn_core_root, 'param', 'default.json')
+    params = read_params(params_path)
+    params['celsius'] = 37.0
+    params['threshold'] = 0.0
```

Comment:
why do you have to modify this? I'm guessing this doesn't get saved in the `hdf5` file?

Reply:
I just copied specifications that were used when I refactored this script into smaller defined tests. There are some simulations being performed in the subsequent tests that are passed this fixture. I figured those parameters important for simulations?

Reply:
I see ... don't think we ever want to have a temperature other than 37 for humans ... my feeling is that we should hard-code it [here](https://github.com/jonescompneurolab/hnn-core/blob/df99e07272c8e3eba6b9f5647d547948ee7ae129/hnn_core/network_builder.py#L58) or alternatively add an argument to `simulate_dipole`. Doesn't seem to fit with the network object.

cc @rythorpe @ntolley